packages/types-core/src/project/types.ts

```diff
@@ -185,3 +185,12 @@ export interface DictionaryQueryEntry {
   entity: string;
   conditions: DictionaryQueryCondition[];
 }
+
+export interface IBlock<B = any> {
```

Comment:
I think this would live under node-core as its an internal type that users shouldn't be aware of